src/components/GameCard.tsx

```diff
@@ -22,12 +25,34 @@ interface GameCardProps {
     notes?: string;
     platformsOwnedOn?: Array<string>;
     dateAdded?: Date;
+}
 
+interface LibraryOption {
+    label: string,
+    value: string
 }
 
 export function GameCard(gameCardProps : GameCardProps ) {
-    // will eventually be an API function
-    const addToLibrary = () => true;
+    const [libraryOptions, setAllLibraryOptions] = useState<Array<LibraryOption>>([])
+    const [selectedLibrary, setSelectedLibrary] = useState("")
+
+    useEffect(() => {
+        const fetchAllLibraries = async () => {
+            const libraries = await libraryApi.getAllLibraries()
+            const libraryOptions = new Array<LibraryOption>()
+            libraries.forEach((library) => {
+                libraryOptions.push({ label: library.name, value: library.id })
+            } )
+            setAllLibraryOptions(libraryOptions)
+        }
+        fetchAllLibraries()
+    }, [])
+
+    const addToLibrary = async (gameId: string, libraryId: string) => {
+        // todo: need to check if it's an instance or game
```

Comment:
Do we have a ticket for this?

Reply:
No explicit ticket, but I think once we have users in place we can have a better idea of how this should be reasoned out. I'm not sure yet how to phrase the ticket

Reply:
actually just kidding, added it here: https://gaming-backlog.atlassian.net/browse/GB-80